pkg/restore/pipeline_items.go

```diff
@@ -159,42 +204,81 @@ func NewTiKVSender(
 			rejectStoreMap[store.GetId()] = true
 		}
 	}
+	inCh := make(chan DrainResult, 1)
+	midCh := make(chan DrainResult, splitConcurrency)
```

Comment:
It seemed not pipeline enough, if we set channel buffer to 1 🤔️

Reply:
Yes, let's make it bigger.